### ISSUE TITLE
F-Droid?

### ISSUE THREAD

**Issue by namazso** [Rouji/Hupl#17]
it would be great to see this app on F-droid

**Comment by Rouji** [comment]
Agreed. It's definitely far up on the TODO list for when I have time/energy to work on this again.

**Comment by quite** [comment]
I just found Hupl, it's really neat. <3 f-droid

**Comment by Rouji** [comment]
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/45207 submitted

**Comment by Rouji** [comment]
https://f-droid.org/packages/eu.imouto.hupl/ :)
